automatic/chromium/tools/chocolateyInstall.ps1

```diff
@@ -15,7 +15,7 @@ if (Test-Path $Chromium) {
 
 $packageArgs = @{
   packageName   = 'chromium'
-  file          = "$toolsdir\chromium_x32.exe"
+  url           = "https://chromium_x32.exe"
```

Comment:
```suggestion
  url           = 'https://chromium_x32.exe'
```

We typically use single quotes in these variables if we do not need to reference another variable.